websites/J/jut.su/dist/metadata.json

```
@@ -11,7 +11,7 @@
 		"nl": "Jut.su is een Russische anime-website"
 	},
 	"url": "jut.su",
-	"version": "1.0.11",
+	"version": "1.1.0",
```

Should be a major bump as you changed the Client ID 
```suggestion
	"version": "2.0.0",
```